qemu/tests/x86_cpu_model.py

```diff
@@ -2,37 +2,11 @@
 import re
 import json
 
-from avocado.utils import cpu, process
-from virttest import error_context, utils_misc, env_process
-
+from avocado.utils import cpu
+from avocado.utils import process
 
-def check_flags(params, flags, test, session=None):
-    """
-    Check cpu flags on host or guest.(only for Linux now)
-    :param params: Dictionary with the test parameters
-    :param flags: checked flags
-    :param test: QEMU test object
-    :param session: guest session
-    """
-    cmd = params["check_flag_cmd"]
-    func = process.getoutput
-    if session:
-        func = session.cmd_output
-    out = func(cmd).split()
-    missing = [f for f in flags.split() if f not in out]
-    if session:
-        error_context.context("Check cpu flags inside guest", logging.info)
-        if missing:
-            test.fail("Flag %s not in guest" % missing)
-        no_flags = params.get("no_flags")
-        if no_flags:
-            err_flags = [f for f in no_flags.split() if f in out]
-            if err_flags:
-                test.fail("Flag %s should not be present in guest" % err_flags)
-    else:
-        error_context.context("Check cpu flags on host", logging.info)
-        if missing:
-            test.cancel("This host doesn't support flag %s" % missing)
+from provider.cpu_utils import check_flags
```

Comment:
import order: build-in ->avocado -> virttest -> tp-qemu, so move this line down.

Reply:
> import order: build-in ->avocado -> virttest -> tp-qemu, so move this line down.

update and thanks.